code/modules/mob/living/carbon/xenomorph/login.dm

```diff
@@ -4,9 +4,8 @@
 		deltimer(afk_timer_id)
 		afk_timer_id = null
 
-	if(see_in_dark == XENO_NIGHTVISION_ENABLED)
-		lighting_alpha = LIGHTING_PLANE_ALPHA_INVISIBLE
-		see_in_dark = XENO_NIGHTVISION_ENABLED
+	if(lighting_alpha == LIGHTING_PLANE_ALPHA_NV_TRAIT)
+		see_in_dark = 8
```

Comment:
```suggestion
		see_in_dark = XENO_NIGHTVISION_ENABLED
```

Reply:
Ah, I see you intentionally removed it.
Shouldn't these be defines, in general?

Reply:
Because it doesn't change. This variable no longer controls whether their vision is enabled or disabled, it stays at 8

Reply:
So replaced with new defines, like the planes and layers?

Reply:
No, it does not need new defines, that would be confusing.